### ISSUE TITLE
force:mdapi:deploy no longer showing error line numbers on failures

### ISSUE THREAD

**Issue by alan-morey** [forcedotcom/cli#356]
### Summary

`force:mdapi:deploy` no longer shows the error line number for components that fail to deploy.

### Steps To Reproduce:
1. Create package folder `deploy-test` with following components:
   ```
      deploy-test/
      ├── classes/
      │   ├── ClassWithCompileError.cls
      │   └── ClassWithCompileError.cls-meta.xml
      └── package.xml
   ```
2. Contents of `ClassWithCompileError.cls`:
    ```apex
   public class ClassWithCompileError {
       privtae String text;
    ```
3. Contents of `ClassWithCompileError.cls-meta.xml`:
   ```xml
   <?xml version="1.0" encoding="UTF-8"?>
   <ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
       <apiVersion>48.0</apiVersion>
       <status>Active</status>
   </ApexClass>
   ```
3. Contents of `package.xml`:
    ```xml
   <?xml version="1.0" encoding="UTF-8"?>
   <Package xmlns="http://soap.sforce.com/2006/04/metadata">

     <types>
       <members>ClassWithCompileError</members>
       <name>ApexClass</name>
     </types>

     <version>48.0</version>
   </Package>
    ```
4. Execute deploy command:
   ```sh
   $ sfdx force:mdapi:deploy -d deploy-test -g -w -1
   Job ID | 0Af2h00000AOa8QCAT
   MDAPI PROGRESS | ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ | 0/3 Files

   === Component Failures [3]
   TYPE   FILE                                           NAME                   PROBLEM
   ─────  ─────────────────────────────────────────────  ─────────────────────  ──────────────────────────────
   Error  deploy-test/classes/ClassWithCompileError.cls  ClassWithCompileError  Unexpected token 'String'.
   Error  deploy-test/classes/ClassWithCompileError.cls  ClassWithCompileError  Expecting '}' but was: '<EOF>'
   Error  deploy-test/classes/ClassWithCompileError.cls  ClassWithCompileError  Unexpected token 'String'.

   ERROR running force:mdapi:deploy:  The metadata deploy operation failed.
   ```

### Expected result

Expected error messages in "Component Failures" list would contain the line/column number where the problem occurs, so that I can easily fix the issue and re-deploy:

```
 === Component Failures [3]
   TYPE   FILE                                           NAME                   PROBLEM
   ─────  ─────────────────────────────────────────────  ─────────────────────  ──────────────────────────────
   Error  deploy-test/classes/ClassWithCompileError.cls  ClassWithCompileError  Unexpected token 'String'. (2:13)
   Error  deploy-test/classes/ClassWithCompileError.cls  ClassWithCompileError  Expecting '}' but was: '<EOF>' (3:1)
   Error  deploy-test/classes/ClassWithCompileError.cls  ClassWithCompileError  Unexpected token 'String'. (2:13)
   ```
_Note the line/column at the end of each message._

### Actual result

No line/column numbers are shown in the error messages making it more difficult for me to track the source of the problem.

### Additional information

I also tried using the `--verbose` flag but this does not add the line/column numbers either.

Using `--json` flag I can see the line/column numbers are present.

As a workaround, I am opening "Deployment Status" page in Salesforce to view errors but really should get that information in `force:mdapi:deploy` without having to use `--verbose` or `--json` flags.

**SFDX CLI Version**: sfdx-cli/7.52.0-c5ad5c3eff linux-x64 node-v10.15.3                                                                                                                             

**SFDX plugin Version**:
```@oclif/plugin-autocomplete 0.1.5                                                                                                                                               
@oclif/plugin-commands 1.2.3 (core)
@oclif/plugin-help 2.2.3 (core)
@oclif/plugin-not-found 1.2.3 (core)
@oclif/plugin-plugins 1.7.9 (core)
@oclif/plugin-update 1.3.9 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-trust 3.0.7 (core)
analytics 1.7.1 (core)
generator 1.1.2 (core)
salesforcedx 48.7.1 (core)
├─ @salesforce/sfdx-plugin-lwc-test 0.1.5 (core)                                                                                                                               
├─ salesforcedx-templates 48.7.0 (core)                                                                                                                                        
└─ salesforce-alm 48.8.0 (core) 
```

**OS and version**: Ubuntu 18.04.4 LTS

**Comment by clairebianchi** [comment]
@alan-morey We will look into it. Out of curiosity if you use source:deploy do you see the line numbers?

**Comment by alan-morey** [comment]
@clairebianchi Yes, line/column numbers appear on failure messages when using `source:deploy`

**Comment by clairebianchi** [comment]
Great, Thank you

**Comment by forcebewithyou** [comment]
Hello, any idea on when this is targeted to be fixed? Thank you so much.

**Comment by dcinzona** [comment]
Still an issue, btw...

*** EDIT *** 
Still an issue as of Spring 21 release (2/16/2021)

**Comment by jshackell-sfdc** [comment]
This issue is fixed in version 2.13.9 (Oct 18, 2023) of Salesforce CLI.
